testsuite/smoke-test/src/test_utils.rs

```diff
@@ -51,6 +51,21 @@ pub async fn transfer_coins(
     txn
 }
 
+pub async fn reconfig(
```

Comment:
Who will call reconfig?

Reply:
some new tests :)

it basically allows you to trigger an epoch change